src/mpid/ch4/netmod/ucx/ucx_init.h

```diff
@@ -197,7 +197,12 @@ static inline int MPIDI_NM_mpi_init_hook(int rank,
                 &MPIDI_UCX_AV(&MPIDIU_get_av(0, i)).dest);
 
         MPIDI_UCX_CHK_STATUS(ucx_status);
-    }
+    /* Actually, It sould not be neccessary to call this here. But due to the current wireup/
+    * diconnect protocoll in UCX, we regular end up in deadlocks/non-flushed EP when we have a lot of
+    * processes and oversusucription without calling this here...  */
+       ucx_status =  ucp_worker_flush(MPIDI_UCX_global.worker); 
```

Comment:
Can we submit a bug to UCX for this issue? I'd prefer we have a FIXME comment covering all the places where we add flushes that just points to the issue in the UCX repo.